lib/exercises.js

```diff
@@ -1,19 +1,54 @@
 // This method will return an array of arrays.
 // Each subarray will have strings which are anagrams of each other
-// Time Complexity: ?
-// Space Complexity: ?
+// Time Complexity: O(N^2)
```

Comment:
✨If you want to include sorting the strings in your time complexity, we could say that the time complexity is O(n m log(m)) where m is the length of the string you are sorting and n is the length of `strings`

Space complexity would also be O(n) where counter and grouped are each of size n